### PR TITLE
docs: Extend aws configuration docs

### PR DESCRIPTION
**Pull request by miguelsorianod**
## Description

* Extends the documentation around setting up Fleet Manager to interact with AWS

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

File: docs/populating-configuration.md [hunk 1]
```diff
@@ -66,8 +66,37 @@ to test things out.
...
+
+To accomplish the previously mentioned functionalities Fleet Manager needs to
+be configured to interact with the AWS account. To do so, provide existing AWS
+IAM user credentials to the control plane by running:
+```
+AWS_ACCOUNT_ID=<aws-account-id> \
+AWS_ACCESS_KEY=<aws-iam-user-access-key> \
+AWS_SECRET_ACCESS_KEY=<aws-iam-user-secret-access-key> \
+ROUTE53_ACCESS_KEY=<aws-iam-user-for-route-53-access-key> \
+ROUTE53_SECRET_ACCESS_KEY=<aws-iam-user-for-route-53-secret-access-key> \
```
Comment (machi1990) [inline docs/populating-configuration.md:99]: We could link to the https://github.com/bf2fc6cc711aee1a0c2a/fleet-manager-go-template/blob/f11f2d5ec106dfa99bba03f8d5f0b1fd031b2af9/docs/getting-credentials-and-accounts.md#aws guide on how to get the credentials.

File: docs/populating-configuration.md [hunk 2]
```diff
@@ -66,8 +66,37 @@ to test things out.
 >NOTE: You could pass dummy values to make sure that the secrets files are created.
 
 ## Setup AWS configuration
-This is used to initialise the aws accounts for cluster creation via OCM and also to setup AWS route53 account.
+Fleet Manager interacts with AWS to provide the following functionalities:
+* To be able to create and manage Data Plane clusters in a specific AWS account
```
[the remark this answers is not in the document]
Comment (miguelsorianod) [reply docs/populating-configuration.md:70]: Updated :+1:

File: docs/populating-configuration.md (hunk 1 above)
Comment (miguelsorianod) [reply docs/populating-configuration.md:100]: Added a note with that. That information is only relevant to Red Hat people so I tried to be clear with that

**Review by machi1990 — APPROVED** [review]

File: docs/populating-configuration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (machi1990) [reply docs/populating-configuration.md:70]: Thanks

File: docs/populating-configuration.md (hunk 1 above)
Comment (machi1990) [reply docs/populating-configuration.md:100]: Thanks

**Pull request merged**